### PR TITLE
[DOCS] add docs for high level client get method

### PR DESCRIPTION
**Pull request by javanna**
This PR adds documentation for the high level client get method.

### CODE REVIEW

File: client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java [hunk 1]
```diff
@@ -417,4 +425,171 @@ public void onFailure(Exception e) {
...
+            //tag::get-request-no-source
+            request.fetchSourceContext(new FetchSourceContext(false)); // <1>
+            //end::get-request-no-source
+            GetResponse getResponse = client.get(request);
+            assertNull(getResponse.getSourceInternal());
+        }
+        {
+            GetRequest request = new GetRequest("index", "type", "id");
+            //tag::get-request-source-include
+            FetchSourceContext fetchSourceContext = new FetchSourceContext(true,
```
Comment (tlrx) [inline client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java]: For a better readability, could we have something like:
```
String[] includes = ...
String[] excludes = ...
FetchSourceContext fetchSourceContext = new FetchSourceContext(true, includes, excludes)
...
```

File: client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java [hunk 2]
```diff
@@ -417,4 +425,171 @@ public void onFailure(Exception e) {
...
+            GetResponse getResponse = client.get(request);
+            Map<String, Object> sourceAsMap = getResponse.getSourceAsMap();
+            assertEquals(2, sourceAsMap.size());
+            assertEquals("trying out Elasticsearch", sourceAsMap.get("message"));
+            assertTrue(sourceAsMap.containsKey("postDate"));
+        }
+        {
+            GetRequest request = new GetRequest("index", "type", "id");
+            //tag::get-request-source-exclude
+            FetchSourceContext fetchSourceContext = new FetchSourceContext(true,
```
Comment (tlrx) [inline client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java]: Same here

**Review by tlrx — APPROVED** [review]
LGTM, left minor comments

File: client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java (hunk 1 above)
Comment (javanna) [inline client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java]: good idea

**Pull request merged**